modules/fakemeta/fakemeta_amxx.cpp

```diff
@@ -39,13 +42,27 @@ void OnAmxxAttach()
 
 	ConfigManager = MF_GetConfigManager();
 
-	char error[256];
-	error[0] = '\0';
+	char error[256] = "";
 
 	if (!ConfigManager->LoadGameConfigFile("common.games", &CommonConfig, error, sizeof(error)) && error[0] != '\0')
 	{
 		MF_Log("Could not read common.games gamedata: %s", error);
-		MF_Log("get/set/find_ent_data* natives have been disabled");
+		MF_Log("get/set_ent_data* natives have been disabled");
```

Comment:
I just think there is no need to spread out error messages like this over multiple lines.